FEM/pcs_dm.cpp

```diff
@@ -772,21 +772,18 @@ double CRFProcessDeformation::Execute(int loop_process_number)
 
 				Error = Norm / InitialNorm;
 				ErrorU = NormU / InitialNormU0;
-				// Error = Norm / InitialNorm;
-				// ErrorU = NormU / InitialNormU0;
-				// if(Norm < Tolerance_global_Newton && Error > Norm)
-				//	Error = Norm;
-				//           if(Norm<TolNorm)  Error = 0.01*Tolerance_global_Newton;
-				// if((NormU / InitialNormU) <= Tolerance_global_Newton)
-				//	Error = NormU / InitialNormU;
+				if(Norm < Tolerance_global_Newton && Error > Norm)
+				    Error = Norm;
+
+				if((NormU / InitialNormU) <= Tolerance_global_Newton)
+					Error = NormU / InitialNormU;
 
```

Comment:
First, Error is a relative residual error, then it can become an absolute residual error, then it can become a relative unknown increment error. In the end, I have no control over why the algorithm converges. This was the reason why I removed this, as I loose control over the chosen measure of accuracy by this.